downstream/modules/automation-dashboard/proc-upgrading-automation-dashboard.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+// assembly-view-key-metrics.adoc
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-upgrading-automation-dashboard"]
+
+= Upgrading {AutomationDashboardName}
+
+[role="_abstract"]
+Upgrade your {AutomationDashboardName} installation to the latest version. This process involves running the installation program and updating your cluster configuration with new authentication requirements.
+
+.Procedure
+
+. Download the latest installation bundle from access.redhat.com. Navigate to Downloads > Red Hat Ansible Automation Platform Product Software.
+. Extract the bundle to a new directory.
+. Copy the `inventory` file from your previous installation directory to the new directory.
++
+[NOTE]
+====
+You can reuse your existing `inventory` file because this upgrade does not introduce any new inventory variables.
+====
```

Comment:
Sorry, I was wrong. There is a new text in inventory that is mandatory - related to redis.

```
[redis]
host.example.com ansible_connection=local

[all:vars]
redis_mode=standalone
```